troubleshooting/ipad-multitasking-support-ios-publish-to-fail.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Invalid Bundle Error When Publishing to the App Store
+description: "Publishing to the Apple App Store fails with an ERROR ITMS-90474: Invalid Bundle. iPad Multitasking support requires these orientations."
+type: troubleshooting
+page_title: Publishing App With IPad Multitasking Support Requires All Device Orientations to Be Enabled
+slug: ipad-multitasking-support-fail-ios-publish
+position:
+tags: appbuilder, publish
+teampulseid:
+ticketid: 1029434, 1045223
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Publish - App Store</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+</table>
+
+## Description
+When you attempt to publish your app to the Apple App Store, the following error message appears: `ERROR ITMS-90474: "Invalid Bundle`.
```

Comment:
The quotation marks in the message bother me. Are they part of the message? If yes, why are they not closed?